src/app/utils/beta-permissions.ts

```diff
@@ -0,0 +1,31 @@
+import { BasicField, IFieldSchema, IUserSchema } from '../../types'
+
+/**
+ * Array contains all current beta fields. Add or remove field types as the beta
+ * fields change.
+ */
+const BETA_FIELDS = [BasicField.Mobile]
```

Comment:
I think this was my mistake while removing `allowSms`, mobile is no longer a beta field so this array should be empty